js/navigation/base.js

```diff
@@ -10,60 +10,64 @@ define([
 
 (function( $, undefined ) {
 
-	// existing base tag?
-	var baseElement = $( "head" ).children( "base" ),
+	var base,
+
+		// existing base tag?
+		baseElement = $( "head" ).children( "base" ),
+
+		// DEPRECATED as of 1.5.0 and will be removed in 1.6.0. As of 1.6.0 only
+		// base.dynamicBaseEnabled will be checked
+		getDynamicEnabled = function() {
+			if ( $.mobile.dynamicBaseEnabled !== undefined ) {
```

Comment:
Yes, because ```true``` and ```false``` are both valid values for ```dynamicBaseEnabled```. In 1.5.0 ```$.mobile.dynamicBaseEnabled``` is ```undefined``` by default so if I find that it is defined, then I must use the value from the deprecated location even if that value is ```false```.